ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx

```diff
@@ -268,13 +268,16 @@ export const ScenePlayer: React.FC<IScenePlayerProps> = ({
   useEffect(() => {
     function isVrScene() {
       if (!scene?.id || !vrTag) return false;
+      let hasVrTag = false;
 
       scene?.tags.map((tag) => {
         if (vrTag == tag.name) {
-          return true;
+          hasVrTag = true;
+          return;
         }
       });
-      return false;
+
+      return hasVrTag;
```

Comment:
If you replace this with `some` per my previous review, it doesn't have to iterate through all of the tags. `map` is used incorrectly here.

Reply:
I'm just now seeing your comment. I'm unsure how I missed it when I checked GitHub the day after.